includes/sanitizers/class-amp-link-sanitizer.php

```diff
@@ -132,11 +150,20 @@ public function process_links() {
 			}
 
 			$href = $element->getAttribute( 'href' );
-
-			if ( $this->is_frontend_url( $href ) && '#' !== substr( $href, 0, 1 ) ) {
+			$rel  = $element->hasAttribute( 'rel' ) ? trim( $element->getAttribute( 'rel' ) ) : '';
```

Comment:
@todo: handle a case where there is more than one rel, like `rel="not-amphtml nofollow"`. It looks like there can be multiple [space-separated values](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-rel).

Reply:
That's correct. It should rather do something like:

```suggestion
			$rel  = $element->hasAttribute( 'rel' ) ? array_filter( preg_split( '/\s+/' $element->getAttribute( 'rel' ) ) ) : [];
```

And then check:

```php
$pos = array_search( self::REL_VALUE_NON_AMP_TO_AMP, $rel );
if ( false !== $pos ) {
    unset( $rel[ $pos ] );
    if ( empty( $rel ) ) {
        $element->removeAttribute('rel');
    } else {
        $element->setAttribute('rel', implode( ' ', $rel ));
    }
}
```

Reply:
Thanks, https://github.com/ampproject/amp-wp/pull/4146/commits/8a2e3dadf769d403b9b1e52866d1c7aab8d84a7d uses these snippets and adds some test cases.